posts/2021-02-15-eclipse-transformer.adoc

```diff
@@ -0,0 +1,506 @@
+---
+layout: post
+title: "Using the Eclipse Transformer for Jakarta EE 9 Development and Testing"
+categories: blog
+author_picture: TBD
+author_github: https://github.com/tbitonti
+seo-title: Using the Eclipse Transformer for Jakarta EE 9 Development and Testing
+seo-description: The Eclipse Transformer, an Eclipse hosted open source project, provides a tool which may be used to quickly create Jakarta EE 9 enabled artifacts from existing Java EE artifacts.  Artifacts which may be transformed include applications in source and binary form, test code, and Open-Liberty server configurations.  This article provides a brief introduction to the Eclipse Transformer and walks through the use of the Eclipse Transformer command line interface.
+blog_description: "The Eclipse Transformer, an Eclipse hosted open source project, provides a tool which may be used to quickly create Jakarta EE 9 enabled artifacts from existing Java EE artifacts.  Artifacts which may be transformed include applications in source and binary form, test code, and Open-Liberty server configurations.  This article provides a brief introduction to the Eclipse Transformer and walks through the use of the Eclipse Transformer command line interface."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Using the Eclipse Transformer for Jakarta EE 9 Development and Testing
+Thomas Bitonti <https://github.com/tbitonti>
+
+== Overview
+
+The core problem introduced by Jakarta EE 9 is the renaming of packages from "javax" to "jakarta".  As a consequence, existing code which runs against Java EE does not run against Jakarta EE 9.  To run existing Java EE applications as Jakarta EE 9 applications, references to Java EE packages which start with "javax" must be updated.  (Note that java packages outside of Java EE that use the initial name "javax".  Only specific Java EE packages should be updated.)
```

Comment:
I think you can remove packages from API and just have APIs like I have above.  The package word is redundant since we include it as part of package prefixes.  

So you decided to remove the note completely?

Reply:
Hi; the note seemed out of place, and less necessary with the addition of "specific" to the preceding sentence.
I've further adjusted this paragraph, per your added comment.